hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmDBAccessIdInfo.java

```diff
@@ -42,16 +42,19 @@
    * Only effective if isAdmin is true.
    */
   private final boolean isDelegatedAdmin;
-
-  // This implies above String fields should NOT contain the split key.
-  public static final String SERIALIZATION_SPLIT_KEY = ";";
+  /**
+   * Role name of the user (that this access ID is assigned to) in this tenant.
+   * e.g. "user", "admin", "auditor"
+   */
+  private final String roleId;
 
   public OmDBAccessIdInfo(String tenantId, String userPrincipal,
-                          boolean isAdmin, boolean isDelegatedAdmin) {
+      boolean isAdmin, boolean isDelegatedAdmin, String roleId) {
     this.tenantId = tenantId;
     this.userPrincipal = userPrincipal;
     this.isAdmin = isAdmin;
     this.isDelegatedAdmin = isDelegatedAdmin;
+    this.roleId = roleId;
```

Comment:
what if accessID has to be part of multiple roles ?

Reply:
Thanks @prashantpogde . Very good point. I have addressed the issue by making the protobuf a `repeated` one.